packages/@react-aria/button/src/useButton.ts

```diff
@@ -54,6 +54,8 @@ export function useButton(props: AriaButtonProps<ElementType>, ref: RefObject<an
     onPressChange,
     // @ts-ignore - undocumented
     preventFocusOnPress,
+    // @ts-ignore - undocumented
+    allowFocusWhenDisabled,
```

Comment:
😬

Reply:
Let's hope no one abuses this to put tooltips on disabled things...

Reply:
why would you call it out then? =P we can always just change the name of the prop every couple releases to really discourage it, hahaha